packages/odata-common/src/de-serializers/de-serialization-middleware.ts

```diff
@@ -0,0 +1,84 @@
+import { identity, isNullish } from '@sap-cloud-sdk/util';
+import BigNumber from 'bignumber.js';
+
+export interface DeSerializationMiddleware<
```

Comment:
for the other `deserializer` we did not split the `de` in kebap and camel case. I would like to have in everywhere the same. I find this `de-serialzer` and DeSeri..... a bit ugly so I would undo it here.

Reply:
I guess this `DeSerialize` means `(de)serialize`, which is a combination of both `serialize` and `deserialize` as one word.

Reply:
hm, yes you are right. But I did not get it from the name.